product_subscription/__openerp__.py

```diff
@@ -1,41 +1,28 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Business Open Source Solution
-#    Copyright (C) 2013-2016 Coop IT Easy SCRL.
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Houssine Bakkali <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
@robinkeunen you may add your name here now

Reply:
![](https://media.giphy.com/media/11Un64tF2yX8JO/giphy.gif)

Reply:
ahah